abstractions/README.md

```
@@ -0,0 +1,21 @@
+# <img src="assets/NSS-128x128.png" align="left" />Nefarius.Vicius.Abstractions
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)